resources/lib/streamservice.py

```diff
@@ -216,6 +216,10 @@ def get_stream(self, video, roaming=False, api_data=None):
             # Get stream manifest url
             manifest_url = next(stream.get('url') for stream in stream_json.get('targetUrls') if stream.get('type') == protocol)
 
+            # External virtual subclip support (live2vod airdate feature)
+            if video.get('start_date') and video.get('end_date'):
+                manifest_url += '?t=' + video.get('start_date') + '-' + video.get('end_date')
```

Comment:
Nice !

Reply:
Unfortunately, the livestream is only archived for 24 hours. I edited the comment accordingly.